File: Problemas_python/problema5.py
# Problema 5: Determinar si una persona debe pagar impuestos
age = int(input("Ingrese su edad: "))
ingresos_mensuales = float(input("Ingrese sus ingresos mensuales: "))

ingresos_anuales = ingresos_mensuales * 12 # Los ingresos totales anuales sean => $65.891.000

if age >= 18 and ingresos_anuales >= 65891000: #pagan solo mayores de edad 
    print("Debido a sus ingresos anuales y edad debe pagar impuestos.")
else:
    print("No debe pagar impuestos.")
    

File: problemas_haskell/problema5.hs
-- Problema 5: Determinar si una persona debe pagar impuestos

debePagarImpuestos :: Int -> Double -> Bool
debePagarImpuestos age ingresosMensuales =
    age >= 18 && (ingresosMensuales * 12) >= 65891000

main :: IO ()
main = do
    putStrLn "Ingrese su edad:"
    edadStr <- getLine
    putStrLn "Ingrese sus ingresos mensuales:"
    ingresosStr <- getLine
    let edad = read edadStr :: Int
        ingresos = read ingresosStr :: Double
    if debePagarImpuestos edad ingresos
        then putStrLn "ebido a sus ingresos anuales y edad debe pagar impuestos."
        else putStrLn "No debes pagar impuestos. "